docs/meshstack.index.md

```diff
@@ -39,3 +39,14 @@ The connections are shown in the following matrix table:
 |      AWS      |                    -                    |                Account                |              IAM Role              |      CF StackSets / Organization Units      |
 |     Azure     |                    -                    |             Subscription              |              AAD User              |        Blueprints / Management Groups        |
 |      GCP      |                    -                    |                Project                |              GCD User              |     Organization Policy / GDM Template      |
+
+
+## Operations
+
+meshcloud will typically operate your meshStack installation as a [managed service](./meshstack.managed-service.md) for you. As a managed service, all configuration and validation is done by meshcloud. Nonetheless, we make references to configuration options in the documentation so that operators get a better understanding of meshStack's capabilities. The configuration references also simplify examples and communicate the exact parameters that may need to be supplied by platform operators (e.g. Service Principal credentials).
```

Comment:
why typically? I thought that was the idea behind it. We can also write as a service

Reply:
i have just moved the existing documentation around, I haven't modified any of the original texts. I agree that it makes sense changing smaller defects like this.